cpp/ql/src/experimental/Security/CWE/CWE-552/DangerousUseRename.qhelp

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE qhelp PUBLIC
+  "-//Semmle//qhelp//EN"
+  "qhelp.dtd">
+<qhelp>
+<overview>
+<p>If the rename function did not work correctly, simply overwriting the destination file can lead to errors. Use access to the destination file with permission setting.</p>
+
+</overview>
+
+<example>
+<p>The following example shows the erroneous and corrected ways to work with the rename function.</p>
```

Comment:
Please explain what is erroneous about the erroneous example.

Reply:
The target file is opened without checking the access rights and its existence, in fact the program overwrites this file.